Sources/BraintreePayPalNativeCheckout/BTPayPalNativeTokenizationClient.swift

```diff
@@ -17,14 +17,14 @@ class BTPayPalNativeTokenizationClient {
     }
 
     func tokenize(
-        request: BTPayPalNativeRequest,
+        request: BTPayPalRequest,
         returnURL: String,
         completion: @escaping (Result<BTPayPalNativeCheckoutAccountNonce, BTPayPalNativeError>) -> Void)
     {
 
         let tokenizationRequest = BTPayPalNativeTokenizationRequest(
           request: request,
-          correlationID: request.riskCorrelationID ?? State.correlationIDs.riskCorrelationID ?? ""
+          correlationID: request.riskCorrelationId ?? State.correlationIDs.riskCorrelationID ?? ""
```

Comment:
should we change the capitalization of `riskCorrelationId` in the PayPal module? In our Swift rewrite we tend to capitalize abbreviations if they aren't the first word - `returnURL` and `urlToParse`

Reply:
Good callout - I was just about to add some comments for this! We have a ticket (DTBTSDK-2058) to rename this property as a whole. Now that we are subclassing things for `BraintreePayPal` for this we will be able to clean it up across all of the modules in one go. This is why I also didn't add a changelog for this as we will tackle that as part of the `convert-paypal` feature branch work.